### PR TITLE
Fix Errors#add with SimpleDelegator

### PR DESCRIPTION
**Pull request by ryansch**
Rails is helpfully passing us a SimpleDelegator to warn about deprecations. Sycamore doesn't know how to correctly process a hash wrapped in a delegator and we end up with a malformed tree. I've added a `to_h` to unwrap the delegator.

I've also updated the docker stuff and corresponding buildkite setup.

### CODE REVIEW

File: lib/metaractor/errors.rb [hunk 1]
```diff
@@ -71,18 +71,19 @@ def initialize
...
     def add(error: {}, errors: {}, object: nil)
       trees = []
-      [error, errors].each do |h| 
+      [error, errors].each do |h|
         tree = nil
         if h.is_a? Metaractor::Errors
           tree = Sycamore::Tree.from(h.instance_variable_get(:@tree))
         else
-          tree = Sycamore::Tree.from(h)
+          tree = Sycamore::Tree.from(h.to_h)
```
Comment (ryansch) [inline lib/metaractor/errors.rb:79]: This doesn't work if there's a nested delegate in the hash.

**Review by morganick — COMMENTED** [review]
How do I run the specs for this?

**Comment by ryansch** [comment]
@morganick With the `Deskfile` loaded you can use `rspec` directly.

**Comment by morganick** [comment]
There isn't a desk file for this repo.

**Comment by ryansch** [comment]
@morganick gitignore got me

**Review by morganick — APPROVED** [review]
uh, yes! 👍

**Pull request merged**